scripts/crash_test.py

```
@@ -0,0 +1,167 @@
+    
+    if killed_pid:
+        print(f"💀 Killed worker {killed_pid}")
+        
+        # Check remaining workers
+        remaining_pids = get_worker_pids()
+        print(f"🏭 Remaining workers: {len(remaining_pids)} processes")
+        
+        # Monitor job progress
+        print("📊 Monitoring job recovery...")
+        
+        for i in range(30):  # Monitor for 30 seconds
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)